mlflow/tracking/client.py

```diff
@@ -110,6 +113,14 @@ def __init__(self, tracking_uri: Optional[str] = None, registry_uri: Optional[st
         # is assigned lazily by `MlflowClient._get_registry_client()` and should not be referenced
         # outside of the `MlflowClient._get_registry_client()` method
 
+        # NB: Tracing dependencies like OpenTelemetry SDK is not installed in mlflow-skinny.
```

Comment:
note: We can remove this check if we have decided to include tracing as a part of skinny.